src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -658,8 +658,12 @@ bool QgsPostgresConn::getTableInfo( bool searchGeometryColumnsOnly, bool searchP
 
   QgsDebugMsgLevel( "getting table info from layer registries: " + query, 2 );
   result = LoggedPQexec( "QgsPostgresConn", query );
-  // NOTE: we intentionally continue if the query fails
-  //       (for example because PostGIS is not installed)
```

Comment:
This indicates that it is possible to load tables from PG without postgis installed. Is this still possible?

Reply:
Comment is back.

Reply:
The line below now returns if the query fails, if I'm not reading that wrong. Is this still possible to load non-postgis tables?

Reply:
If the result is not PGRES_TUPLES_OK it won't. 

I can remove the change but it will crash if the connection goes down because r`esult.PQntuples()` will execute on a 0x0 mRes. 

Any idea how to fix this? Do we need a method to check for mRes not nullptr?

Reply:
I'm not sure it breaks something.
Does `PQresultStatus()` return the same on connection lost and no postgis available?

Reply:
Done in last commit.